infra/gcp/bash/ensure-staging-storage.sh

```diff
@@ -401,6 +401,22 @@ function staging_special_case__k8s_staging_cluster_api_gcp() {
     ensure_staging_gcb_builder_service_account "cluster-api-gcp" "k8s-infra-prow-build-trusted"
 }
 
+# In order to build the release artifacts, the group needs to be
+# able to create and manage a keyring to encrypt a secret token
+# that will be accessed and decrypted by a cloud build job.
+function staging_special_case__k8s_staging_kustomize() {
+    readonly STAGING_PROJECT="k8s-staging-kustomize"
+
+    local sa_email_group="k8s-infra-staging-kustomize@kubernetes.io"
+    local principal_group="serviceAccount:${sa_email_group}"
```

Comment:
ahhhhh I just caught this after it merged, I'll open a followup
```suggestion
    local principal_group="group:${sa_email_group}"
```